produce_snp_matrix.py

```diff
@@ -0,0 +1,44 @@
+from glob import glob
+import subprocess
+import argparse
+import os
+
+def snps(input_path, output_path):
+    # glob filenames
+    megafasta = ""
+    paths = glob(input_path + "*_consensus.fas")
+    for path in paths:
+        file = open(path)
+        fasta = file.read()
+        megafasta = megafasta + fasta + '\n'
```

Comment:
could do
`fasta_list = []`
`for path in paths:`
`with open(path) as file: fasta = file.read()`
`fasta_list.append(fasta)`
`megafasta = "\n".join(fasta_list)`

Reply:
Your method is also fine though, this may just a bit faster for lots of samples